server/game/effects.js

```diff
@@ -441,6 +441,21 @@ const Effects = {
             }
         };
     },
+    changeConflictSkillFunction: function(func) {
+        return {
+            apply: function (card, context) {
+                if(context.game.currentConflict) {
+                    context.skillFunction = context.game.currentConflict.skillFunction;
```

Comment:
Minor nitpick, but I'd change the name of `context.skillFunction` to something like `originalSkillFunction`, to make it easy to recognize what it's used for, in case someone later stumbles upon it.

Reply:
I think this is a good idea, and I'm going to change it in my Massing at Twilight implementation.